### DISCUSSION TITLE
How to use API in zero-ui?

### DISCUSSION THREAD

**Discussion by dronmaxman** [dec0dOS/zero-ui#167] [Q&A] [answered]
I have raised a fresh instance using docker. I'm not getting to use the API.
I tried using the token from db.json and zerotier-one/authtoken.secret.

I get a 401 error

```
export TOKEN=041a460c9ed2e674c52d87b63b807e09
export NWID= 3c887e55011964d5
curl "https://zt.mydomain.guru/api/network/${NWID}" -H "X-ZT1-AUTH: ${TOKEN}"
{"error":"Specify token"}%
```

**Comment by dec0dOS** [comment, marked as answer]
Please check if the browser sending the request correctly. You could export any network request to CURL
https://www.scrapingbee.com/tutorials/how-to-extract-curl-requests-from-chrome/

**Reply by dronmaxman** [reply]
Yes, it's work. Thanks!
